docs/engineering/tooling/uploader.mdx

```diff
@@ -0,0 +1,10 @@
+# uploader script
```

Comment:
@Wambere does efsity handle uploading yet or are people still using this script?

Reply:
I think some people are still using the script, but the uploader is ready for use. I'll set up a session for a walk-through just so everyone is aware